tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = flake8, py27, py34, py36
+envlist = flake8, py27, py36
```

Comment:
remove this too: https://github.com/app-sre/qontract-reconcile/blob/master/setup.py#L50

Reply:
done, thanks!